Backend/src/main/java/com/example/Backend/controller/UserController.java

```diff
@@ -0,0 +1,136 @@
+package com.example.Backend.controller;
+
+import com.example.Backend.model.UserLogin;
+import com.example.Backend.model.UserRegister;
+import com.example.Backend.services.LoginService;
+import com.example.Backend.services.RegisterService;
+import jakarta.validation.Valid;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.*;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+public class UserController {
+
+    RegisterService registerService;
+    LoginService loginService;
+
+    UserController(RegisterService registerService, LoginService loginService) {
+        this.registerService = registerService;
+        this.loginService = loginService;
+    }
+
+    @Operation(summary = "Display user details", description = "Get user details by username")
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "400", description = "Empty Username"),
+            @ApiResponse(responseCode = "404",description = "User Doesn't exists"),
+            @ApiResponse(responseCode = "200", description = "User Fetched Successfully")
+    })
+    @GetMapping("/user/{name}")
+    public ResponseEntity<UserLogin> displayUser(@PathVariable(value = "name") String name) {
+        if(name.isEmpty())
```

Comment:
Use StringUtils.isEmpty funtion. it has null value check already present. Refer - https://www.geeksforgeeks.org/string-handling-with-apache-commons-stringutils-class-in-java/